### PR TITLE
 Fixed an issue where caseless=true doesn't work on algorithm=literal.

### PR DESCRIPTION
**Pull request by spearmin10**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
caseless=true doesn't work when using algorithm=literal.

Video:
https://paloaltonetworks.zoom.us/rec/play/cLi7btoTmCtkMD4MOxXjQnIGMAJuxkdGlsI0xlQUgXYof3sh-piViLilISs21_dUwE-O6Zcpwdt9nuyj.oYbQ_E41EUwgHjhb?autoplay=true
Passcode: @4tGtdsQ1

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by tcarmeli1 — CHANGES_REQUESTED** [review]
Hi @spearmin10 ,
Thank you for your contribution!
Good work :)
I guess the type changes are due to pre-commit right?
Can you please reshare the video? i could not manage to log in.
Thank you!

**Comment by spearmin10** [comment]
> Hi @spearmin10 , Thank you for your contribution! Good work :) I guess the type changes are due to pre-commit right? Can you please reshare the video? i could not manage to log in. Thank you!

Hi @tcarmeli1,
Thank you for your review.

Yes, types were automatically changed by pre-commit.
The video link is here and shared for you. I think you should be able to access with PANW SSO login.

Video: https://paloaltonetworks.zoom.us/rec/play/cLi7btoTmCtkMD4MOxXjQnIGMAJuxkdGlsI0xlQUgXYof3sh-piViLilISs21_dUwE-O6Zcpwdt9nuyj.oYbQ_E41EUwgHjhb?autoplay=true
Passcode: @4tGtdsQ1

Thank you,

**Comment by tcarmeli1** [comment]
Hi @spearmin10 ,
I tried to log in to the video but i get an error for invalid passcode.
maybe can you please check again that the video details are correct?
Thank you so much!

**Review by tcarmeli1 — APPROVED** [review]

File: Packs/FiltersAndTransformers/ReleaseNotes/1_2_78.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+
+##### MapPattern
+
+- Fixed an issue where caseless=true doesn't work on algorithm=literal.
```
Comment (ShirleyDenkberg) [inline Packs/FiltersAndTransformers/ReleaseNotes/1_2_78.md:6]:
```suggestion
Fixed an issue where caseless=true doesn't work on algorithm=literal.
```

**Comment by ShirleyDenkberg** [comment]
@tcarmeli1 Doc review completed.

**Review by tcarmeli1 — APPROVED** [review]

**Comment by tcarmeli1** [comment]
Watched the demo video and approved it.

**Pull request merged**